moveit_kinematics/cached_ik_kinematics_plugin/src/measure_ik_call_cost.cpp

```diff
@@ -139,13 +139,13 @@ int main(int argc, char* argv[])
         num_failed_calls++;
       ++i;
       if (i % 100 == 0)
-        ROS_INFO_NAMED("measure_ik_call_cost",
+        ROS_INFO_NAMED("cached_ik_kinematics_plugin",
```

Comment:
Why did you changed the logger name here? In order to include cached_ik, I would suggest `cached_ik.measure_ik_call_cost`. Remember that the package name (`moveit_kinematics`) is always part of the logger name.